tools/reviewer/job/sync/full_github_pr.proto

```diff
@@ -5,55 +5,37 @@ package com.google.startupos.tools.reviewer.job.sync;
 option java_package = "com.google.startupos.tools.reviewer.job.sync";
 option java_outer_classname = "GithubPullRequestProtos";
 
+// TODO: Remove unused fields
 message PullRequest {
 
   int64 id = 1;
   int64 number = 2;
+  // valid values: `open`, `closed`
   string state = 3;
   string title = 4;
   // Pull Request author
   User user = 5;
   string body = 6;
+  // expected format for created_at and updated_at fields:
+  // `yyyy-MM-ddTHH:mm:ssZ`
   string created_at = 7;
   string updated_at = 8;
-  repeated Review reviews = 9;
-  repeated IssueComment issueComment = 10;
-  CommitPointer base = 11;
-  CommitPointer head = 12;
+  repeated ReviewComment review_comment = 9;
+  repeated IssueComment issue_comment = 10;
+  string base_branch_name = 11;
+  string head_branch_name = 12;
   string repo = 13;
   repeated CommitInfo commits_info = 14;
+  string owner = 15;
+  int64 associated_reviewer_diff = 16;
 }
 
-message CommitPointer {
-  string sha = 1;
-  string ref = 2;
-  User user = 3;
-}
-
-message Review {
-  enum State {
-    UNKNOWN = 0;
-    APPROVED = 1;
-    CHANGES_REQUESTED = 2;
-    COMMENTED = 3;
-    PENDING = 4;
+message ReviewComment {
+  message Link {
+    message PullRequestLink { string url = 1; }
```

Comment:
This is not tied to a GitHub API, right?
How about we remove `PullRequestLink` and put the `url` directly under `Link`?

Reply:
This is also tied to GitHub API.
This is an example of how it looks in the response :
```
...
    "_links": {
      "self": {
        "href": "https://api.github.com/repos/octocat/Hello-World/pulls/comments/1"
      },
      "html": {
        "href": "https://github.com/octocat/Hello-World/pull/1#discussion-diff-1"
      },
      "pull_request": {
        "href": "https://api.github.com/repos/octocat/Hello-World/pulls/1"
      }
    }
...
```